docs/guides/node-operator-manual.md

```diff
@@ -178,30 +214,44 @@ Etherscan pages for the Görli/Prater contracts:
 - [`Lido`](https://goerli.etherscan.io/address/0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F#readProxyContract)
 - [`NodeOperatorsRegistry`](https://goerli.etherscan.io/address/0x9D4AF1Ee19Dad8857db3a45B0374c81c8A1C6320)
 
+Etherscan pages for the Mainnet contracts:
+
+- [`Lido`](https://etherscan.io/address/0xae7ab96520de3a18e5e111b5eaab095312d7fe84#readProxyContract)
+- [`NodeOperatorsRegistry`](https://etherscan.io/address/0x55032650b14df07b85bf18a3a3ec8e0af2e028d5#readProxyContract)
+
 [`addsigningkeysoperatorbh` function]: https://github.com/lidofinance/lido-dao/blob/971ac8f/contracts/0.4.24/nos/NodeOperatorsRegistry.sol#L250
 [`getoperators()` function]: https://github.com/lidofinance/lido-dao/blob/971ac8f/contracts/0.4.24/Lido.sol#L361
 [`lib/abi/nodeoperatorsregistry.json`]: https://github.com/lidofinance/lido-dao/blob/971ac8f/lib/abi/NodeOperatorsRegistry.json
 [`nodeoperatorsregistry.getnodeoperator`]: https://github.com/lidofinance/lido-dao/blob/971ac8f/contracts/0.4.24/nos/NodeOperatorsRegistry.sol#L335
 
-#### Using the key submitter UI
+#### Using the batch key submitter UI
+
+Lido has specialised key submitters: [Mainnet web interface for submitting the keys] and a [Testnet web interface for submitting the keys].
+
+It will require that all keys are validated first - click the check button. Please be patient - we aim to process 1k keys under a minute, but several users can be checking keys simultaneously, so the process can take longer. Also, please don't update the page manually - UI will automatically repeat failing requests and will show an error if retries fail.
```

Comment:
Please, validate the keys before submission by clicking the `check` button. We're aiming to check 1k keys / minute, so this could take some time. Please, don't refresh the page during the checks — this would just start the process over from the very beginning. UI will retry failing requests if there are any errors, and will show an error directly if retries fail.